common-features/extensions-ai-integration.md

```diff
@@ -0,0 +1,41 @@
+---
```

Comment:
The file name `extensions-ai-integration` is "partially" comprehensive and SEO-friendly. Either use something like `microsoft-extensions-ai-integration.md`, or if Microsoft is not important, just use `ai-integration.md`. The second option is also valid if you think we will mention other AI packages on this page in the future.